deploy.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+gcloud config set project llvm-build-dashboard &&
+mvn package com.google.cloud.tools:appengine-maven-plugin:deploy
```

Comment:
(An issue needs to be created to create the app.yaml file)

Reply:
#108